content/docs/command-reference/exp/show.md

```diff
@@ -0,0 +1,87 @@
+# exp show
+
+Display `dvc experiments` as a table, with optional formatting.
+
+> Press `q` to exit.
+
+## Synopsis
+
+```usage
+usage: dvc exp show [-h] [-q | -v] [-a] [-T] [--all-commits] [--no-pager]
+                    [--include-metrics <metrics_list>]
+                    [--exclude-metrics <metrics_list>]
+                    [--include-params <params_list>]
+                    [--exclude-params <params_list>]
+                    [--sort-by <metric/param>] [--sort-order {asc,desc}]
+                    [--no-timestamp] [--sha] [--show-json]
+                    [--precision <n>]
+```
+
+## Description
+
+Shows experiments in a detailed table which includes parameters and metrics.
+Only the experiments derived from the Git `HEAD` are shown by default but all
+experiments can be included with the `---all-commits` option.
+
+`dvc exp show` enters a paginated screen by default, which you can exit by
+typing `Q` (or Esc) in your keyboard. Use `--no-pager` to print the entire
+output to terminal instead.
+
+## Options
+
+- `-a`, `--all-branches` - print experiments derived from the tip of all Git
+  branches instead of just those derived from the current workspace. Note that
```

Comment:
> instead of just those derived from the current workspace

Did you mean those derived from `HEAD` (last commit)?

Reply:
Yes it should be those derived from `HEAD`